NEWS.md

```diff
@@ -6,6 +6,10 @@ The following functions are now re-exported from `{insight}` package:
 `object_has_names()`, `object_has_rownames()`, `is_empty_object()`,
 `compact_list()`, `compact_character()`
 
+NEW FUNCTIONS
+
+  * To convert rownames to a column, and vice-versa: `rownames_as_column()` and `column_as_rownames()`.
```

Comment:
In the parentheses, please also mention your GitHub handle and PR number, so `(@etiennebacher, #80)`.

Reply:
should I also mention issue #2 ?

Reply:
Not needed. The PR already contains that information.